gax-java/dependencies.properties

```diff
@@ -75,7 +75,7 @@ maven.org_graalvm_sdk=org.graalvm.sdk:graal-sdk:22.3.1
 
 # Testing maven artifacts
 maven.junit_junit=junit:junit:4.13.2
-maven.org_mockito_mockito_core=org.mockito:mockito-core:2.28.2
+maven.org_mockito_mockito_core=org.mockito:mockito-core:4.11.0
```

Comment:
Is this a required change? If not, I would leave it out of this PR, and create another PR for updating the mockito version.

Reply:
I believe this was failing the CI when running the integration tests. I can upgrade this in a separate PR.